docker/config_template.yaml

```diff
@@ -213,8 +213,10 @@ kafka:
             topic: temporal-visibility-dev
             dlq-topic: temporal-visibility-dev-dlq
 
+{{ $publicIp := default .Env.BIND_ON_IP "127.0.0.1" -}}
+{{- $defaultPublicHostPost := (print $publicIp ":7233") -}}
 publicClient:
-    hostPort: {{ default .Env.BIND_ON_IP "127.0.0.1" }}:7233
+    hostPort: {{ default .Env.PUBLIC_FRONTEND_ADDRESS $defaultPublicHostPost }}
```

Comment:
I would rename `publicClient` to `sdkClient` everywhere. It might be too late though.

Reply:
Let's follow up in another PR here.